test/api/publish.test.js

```diff
@@ -19,6 +20,13 @@ import {
 beforeEach(() => MockStorage.cleanUp());
 afterEach(() => MockStorage.cleanUp());
 
+function verifyAndDecodeJwtToken(res) {
+  const token = res.text;
+  const secret = config.jwtSecret;
+  const decodedPayload = jwt.verify(token, secret, { algorithms: ['HS256'] });
```

Comment:
I wonder if we can redefine the type definition to require an algorithms field. Also, what was your decision process for HS256?

Reply:
1. This is the default.
2. I didn't see any security reason to not use it. As long as we use a good secret it's secure.
3. We don't need to use a public/private key mechanism for our use of JWT tokens, so HS256 makes sense.

To be completely honest I don't exactly comprehend in which cases something like HS384 or HS512 would be necessary.

Reply:
> I wonder if we can redefine the type definition to require an algorithms field.

TBH I was disappointed that the library isn't more opinionated about that nowadays.
To redefine the type definition I believe I'd have to copy it and I don't really want to do that. Instead I'll take care to encapsulate the logic in one place only.